src/model.ts

```diff
@@ -126,10 +167,43 @@ export class Model {
         'AI.MODELGET reply did not had the full elements to build the Model. Missing ' + missingArr.join(',') + '.',
       );
     }
-    const model = new Model(backend, device, [], [], blob);
+    const model = new Model(backend, device, inputs, outputs, blob);
     if (tag !== null) {
       model.tag = tag;
     }
+    if (batchsize !== null) {
+      model.batchsize = batchsize;
+    }
+    if (minbatchsize !== null) {
+      model.minbatchsize = minbatchsize;
+    }
```

Comment:
why not part of constructor?

Reply:
you're right, but will add it as an optional parameter to it, given that most people will not use them at least on a first usage.

Reply:
solved on the new commit